src/siteSettings/components/SiteDetailsSettingsCard/SiteDetailsSettingsCard.tsx

```diff
@@ -59,18 +59,17 @@ const SiteDetailsSettingsCard: React.FC<SiteDetailsSettingsCardProps> = ({
         />
         <FormSpacer />
         <TextField
-          disabled={disabled}
+          disabled={disabled || IS_CLOUD_INSTANCE}
```

Comment:
Should it be disabled? This looks like a standard use case for a read-only input field - we want you to freely copy and see this field just not be able to edit it directly.

Reply:
I've asked @timuric if we should make this field disabled. Initialy for me it looked like a bug, the field looked exactly the same as the rest but the input was ignored. I only figured it out that it was read-only when I looke at the code 😆.

Disabled is more clear to me that "nope, you can't edit that". You can still copy the value if you need to.

I guess we can leave it as read-only but it should be more clear to the user - maybe some label ("This field is read-only") or CSS class that makes it look like disabled but allows cursor (so that the user can do Cmd+A and Cmd+C to copy the entire field).

Reply:
Nevermind we've decided to remove those fields altogether